widgetssdk/src/main/java/com/glia/widgets/call/CallController.java

```diff
@@ -238,6 +238,9 @@ public void onDestroy(boolean retain) {
         }
     }
 
+    public void onPause() {
```

Comment:
unused?

Reply:
will be used after tied with future commit